src/plone/restapi/deserializer/configure.zcml

```diff
@@ -43,7 +43,7 @@
     <adapter factory=".relationfield.RelationChoiceFieldDeserializer" />
   </configure>
 
-  <configure zcml:condition="installed Products.Archetypes">
+  <configure zcml:condition="installed Products.Archetypes.atapi">
```

Comment:
I guess we should add a comment here explaining why we check for ".atapi" in addition to just AT. Otherwise someone could want to "optimize" this again in the future.